capstone/src/main/webapp/matching.js

```diff
@@ -48,15 +48,22 @@ function changeRiderId(driverId, riderName) {
     return fetch('/setRiderId?driverId=' + driverId +"&riderName=" + riderName).then(response => response.json());
 }
 
+function updateSeats(driverId) {
+    return fetch('/update-seats?driverId=' + driverId).then(response => response.json());
+}
+
 //called when user clicks button
 function setId(){
     console.log("entered set Id");
     let newDriverId = document.getElementById("driverid").value
     let curRider = document.getElementById("first").value;
     changeRiderId(newDriverId, curRider)
-    console.log("Succesfully set " + riderName +"'s Id to " + newDriverId );
+    console.log("Succesfully set " + curRider +"'s Id to " + newDriverId );
+    updateSeats(newDriverId);
+    console.log("Updated ammount of seats");
```

Comment:
"amount", but I think perhaps "number" would be even better.

Reply:
Updated